client/src/pages/onboarding/steps/AcceptCredential.tsx

```diff
@@ -86,7 +86,7 @@ export const AcceptCredential: React.FC<Props> = ({ content, connectionId, crede
     CredentialEventTypes.CredentialStateChanged,
     (event: { payload: { credentialRecord: CredentialExchangeRecord } }) => {
       if (event.payload.credentialRecord.connectionId === connectionId) {
-        dispatch(fetchCredentialEventByConnectionId(event.payload.credentialRecord))
+        dispatch(setCredential(event.payload.credentialRecord))
```

Comment:
I think its more `AddCredential`, as you're adding it to a list of credentials here.

Reply:
Got it. On it.